AsyncKeyManipulationSample/app/src/main/java/com/rivetz/rivetsample5/MainActivity.java

```diff
@@ -135,7 +162,15 @@ public void checkKeyExistenceComplete(RivetKeyDescriptor descriptor, Throwable t
     // Creates a Key asynchronously
     public void createKey() {
         try {
-            crypto.createKey("MyKey", RivetKeyTypes.NISTP256, new RivetRules[0]).whenComplete(this::createKeyComplete);
+            // If DRT supported add Dual Root of Trust usage rule
+            RivetRules rules[];
```

Comment:
It's fewer lines of code and cleaner to set rules[] to null, then update it in the if. No else needed. The API explicitly allows the rules parameter to be null, for the simple case where there are no rules.